driver/driver_config_builder.py

```diff
@@ -177,6 +186,10 @@ class DriverConfig(NamedTuple):  # pylint: disable=too-many-instance-attributes
     disable_index_stats: bool
     num_index_to_collect_stats: int
 
+    db_non_default_parameters: List[
+        str
+    ]
```

Comment:
```suggestion
    db_non_default_parameters: List[str]
```

no need for the line breaks I think